src/assets/src/components/addQueue.tsx

```diff
@@ -260,14 +124,14 @@ export function AddQueuePage(props: PageProps) {
     }
 
     // Set up page state
-    const [activeKey, setActiveKey] = useState('general' as availableTabs);
+    const [activeKey, setActiveKey] = useState(AvailableTabs.General as AvailableTabs);
     const [showCorrectGeneralMessage, setShowCorrectGeneralMessage] = useState(false);
```

Comment:
I might be mistaken, but it seems like this could be a variable derived from some other state? Maybe something like `const showCorrectGeneralMessage = nameValidationResult && descriptValidationResult && allowedValidationResult`?

Reply:
So, yes, we could derive this from the validation results. However, that would mean the "Please correct" alert would show up on a change as well as on a submit. The way it's written now, the user has to hit the submit button and then this alert pops up. I think that might be clearer from a UI perspective, since they get a clear response when they first try to hit submit, rather than nothing happening. It's a subtle difference, I guess I would be fine with it either way.

Reply:
I'm good with that, just always looking for ways to pare down state.